cli/test/test-frame-fit-resize.mjs

```
@@ -0,0 +1,96 @@
+import { Box } from 'ink'
+import React from 'react'
+import { Header } from '../src/build/onboarding/ui/components.tsx'
+import { pickPlatformLayout } from '../src/build/onboarding/ui/frame-fit.ts'
+import { PlatformPicker } from '../src/build/onboarding/ui/platform-picker.tsx'
+import { useTerminalSize } from '../src/build/onboarding/ui/shell.tsx'
+import { renderFrameText, renderResizeFrames } from './helpers/frame-fit.mjs'
+
+let passed = 0
+let failed = 0
+async function test(name, fn) {
+  try { await fn(); passed++; console.log(`✔ ${name}`) }
```

ESLint

[error] This line has 4 statements. Maximum allowed is 1.

(style/max-statements-per-line)